chalice/deploy/packager.py

```diff
@@ -499,6 +499,7 @@ def _calculate_name_and_version(self):
             # {distribution}-{version}(-{build tag})?-{python tag}-{abi tag}-
             # {platform tag}.whl
             name, version = self.filename.split('-')[:2]
+            name = name.replace('_', '-')
```

Comment:
It might be smart to use the PEP 503 normalization rules here, or at least in the ``__eq__()`` implementation.

Reply:
Ill update the commit and look at PEP 503